wpilibj/src/main/java/edu/wpi/first/wpilibj/IterativeRobotBase.java

```diff
@@ -40,6 +40,8 @@
  *   - testPeriodic()
  */
 public abstract class IterativeRobotBase extends RobotBase {
+  protected double m_period;
```

Comment:
What happens if a user derives directly from IterativeRobotBase and doesn't initialize m_period?  I think it will error every message loop.  Seems like we should check that m_period != 0 in the loop.

Reply:
I could make m_period private and make the period a constructor parameter instead of providing setPeriod(). The user shouldn't have to change the period after the robot has initialized.